### ISSUE TITLE
[Birthday] Exception upon Birthday List

### ISSUE THREAD

**Issue by KamiCreed** [SFUAnime/Ren#49]
### Type:

- [ ] Suggestion
- [x] Bug

### Brief description of the problem
`renbirthday list` -> exception.
### Expected behavior

Show list of birthdays

### Actual behavior

```
[21/03/2018 00:57] ERROR red on_command_error 369: Exception in command 'birthday list'
Traceback (most recent call last):
  File "lib/discord/ext/commands/core.py", line 50, in wrapped
    ret = yield from coro(*args, **kwargs)
  File "/home/ren/github/Ren/cogs/birthday.py", line 229, in _birthdayList
    text = "{0:%B} {0:%d}: {1}".format(userBirthday, userObject.name)
AttributeError: 'NoneType' object has no attribute 'name'
```

**Comment by Injabie3** [comment]
Ah, there is currently no catch for checking if the user has left the server.  Thanks for the head's up!

**Comment by Injabie3** [comment]
Fixed in 6f880691b8cc808dfef1c070d499125865e117b0
